src/nisarqa/utils/lonlat.py

```diff
@@ -2,52 +2,93 @@
 
 import os
 import textwrap
-from collections.abc import Sequence
+from collections.abc import Generator, Iterable, Sequence
 from dataclasses import dataclass
 from pathlib import Path
+from typing import Any
+
+import numpy as np
 
 import nisarqa
 
 objects_to_skip = nisarqa.get_all(__name__)
 
 
-def unwrap_longitudes(lon_lat_points: Sequence[LonLat]) -> list[LonLat]:
+def unwrap_longitudes(lons: Iterable[float]) -> list[float]:
     """
-    Normalize longitudes so that they form a continuous quadrilateral
-    across the antimeridian.
+    Unwrap longitudes if the values cross the antimeridian.
+
+    Specifically, longitudes are normalized so that the absolute difference
+    between any adjacent pair of longitude values is <= 180 degrees.
+
+    Of note, in the case of the an antimeridian crossing, longitude values
+    are "unwrapped" to extend beyond the interval of +/-180 degrees
+    for the crossing.
```

Comment:
Your description in `normalize_lon_lat_pts` below is very good. Let's use that same description here as well.

```suggestion
    Specifically, longitudes are normalized so that the absolute difference
    between any adjacent pair of longitude values is <= 180 degrees.

    All given longitude values are first wrapped to +/-360 degrees, and
    then in the case of the an antimeridian crossing they are "unwrapped"
    to extend beyond the interval of +/-180 degrees for the crossing.
```